terminus-op/templates/tdb-statefulset.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Release.Name}}-sts-{{.Release.Namespace}}
+spec:
+  selector:
+    matchLabels:
+      app: {{.Release.Name }}-sts-{{.Release.Namespace }}
+  serviceName: {{.Release.Name}}-svc-{{.Release.Namespace}}
+  replicas: {{ .Values.replicaCount}}
+  template:
+    metadata:
+      labels:
+        app: {{.Release.Name }}-sts-{{.Release.Namespace }}
+    spec:
+      containers:
+      - name: terminus-db-{{.Release.Name}}
+        image: {{ .Values.statefulset.image.name}}

```

Comment:
This should include the tag as well I think.

Reply:
I committed that change , pushing it now!